modules/tracker/mbt/include/visp3/mbt/vpMbGenericTracker.h

```diff
@@ -126,6 +126,7 @@ class VISP_EXPORT vpMbGenericTracker : public vpMbTracker
   virtual std::vector<std::string> getCameraNames() const;
 
   using vpMbTracker::getCameraParameters;
+  virtual void getCameraParameters(vpCameraParameters &camera) const;
```

Comment:
It is defined in `vpMbTracker` parent class. Quick workaround is to redefine it in the `vpMbGenericTracker` class.